op-node/rollup/derive/types_test.go

```diff
@@ -0,0 +1,52 @@
+package derive
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestIsBrotli(t *testing.T) {
```

Comment:
nit, this tests also `ValidCompressionAlgoType` so maybe 
```suggestion
func TestCompressionAlgo(t *testing.T) {
```